Src/Support/Google.Apis.Auth/GoogleJsonWebSignature.cs

```diff
@@ -178,15 +201,45 @@ internal SignedTokenVerificationOptions ToVerificationOptions()
         public static Task<Payload> ValidateAsync(string jwt, ValidationSettings validationSettings) =>
             ValidateInternalAsync(jwt, validationSettings);
 
+        /// <summary>
+        /// Validates a Google-issued Json Web Token (JWT).
+        /// Will throw a <see cref="InvalidJwtException"/> if the specified JWT fails any validation check.
+        /// </summary>
+        /// <remarks>
+        /// <para>Follows the procedure to
+        /// <see href="https://developers.google.com/identity/protocols/OpenIDConnect#validatinganidtoken">validate a JWT ID token</see>.
+        /// </para>
+        /// <para>
+        /// Issued-at validation and expiry validation is performed using the clock on this local client,
+        /// so local clock inaccuracies can lead to incorrect validation results.
+        /// Use <see cref="ValidationSettings.IssuedAtClockTolerance"/> and <see cref="ValidationSettings.ExpirationTimeClockTolerance"/>
+        /// to allow for local clock inaccuracy
+        /// <c>IssuedAtClockTolerance</c> defaults to 30 seconds; it is very unlikely a JWT will be issued that isn't already valid.
+        /// <c>ExpirationTimeClockTolerance</c> defaults to zero seconds; in some use-cases it may be useful to set this to a negative
+        /// value to help ensure that passing local validation means it will pass server validation.
+        /// Regardless of whether local validation passed, code must always correctly handle an invalid JWT error
+        /// from the server.
+        /// </para>
+        /// <para>Google certificates are cached, and refreshed once per hour. This can be overridden by setting
+        /// <see cref="ValidationSettings.ForceGoogleCertRefresh"/> to true.</para>
+        /// </remarks>
+        /// <param name="jwt">The JWT to validate.</param>
+        /// <param name="validationSettings">Specifies how to carry out the validation.</param>
+        /// <param name="cancellationToken">Cancellation token</param>
+        /// <returns>The payload of the verified token.</returns>
+        /// <exception cref="InvalidJwtException">If the token does not pass verification.</exception>
+        public static Task<Payload> ValidateAsync(string jwt, ValidationSettings validationSettings, CancellationToken cancellationToken) =>
+            ValidateInternalAsync(jwt, validationSettings, cancellationToken);
+
         // internal for testing
-        internal static async Task<Payload> ValidateInternalAsync(string jwt, ValidationSettings validationSettings)
+        internal static async Task<Payload> ValidateInternalAsync(string jwt, ValidationSettings validationSettings, CancellationToken cancellationToken = default)
```

Comment:
Better to be explicit in our own internal code.

```suggestion
        internal static async Task<Payload> ValidateInternalAsync(string jwt, ValidationSettings validationSettings, CancellationToken cancellationToken)
```

Reply:
After removing the default value I had to create a version of `ValidateInternalAsync` without a token in order for the tests to build. An alternative would be to alter all tests.

Reply:
Yes please, alter all the tests. Thanks!